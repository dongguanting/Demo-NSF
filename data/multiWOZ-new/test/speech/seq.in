i 'd like to find a train that leaves monday an arrives bye nineteen hundred hours
i need to leave from peter buro
just one please
so i would like to leave from stanstead airport after eleven forty five
i 'd liked to dip art from stevenage after nineteen forty five
i 'm looking for train leaving on tuesday please
actually i need to go to london liverpool street from cab bridge
so i would like to leave anytime after twenty fifteen
i 'm looking for a train going to cambridge
i would like to leave brock  's born on tuesday
i 'd just like to arrive by 16:45 please
yes i would like a ticket for wun
i 'd like a train to cambridge that arrives by 11:30
i 'm deporting from e lee
im head to leicester from cambridge on saturday
i meant to say i wanted to leave after twenty forty five
i one to arrive by nineteen thirty please
i am looking for a train departing from london kings cross
yes dip parting from their heading to cambridge on saturday
the train should go to cambridge and should arrive by 14:00
plea 's book that for five people
i i am traveling on monday and need to leave after sixteen thirty
i need to arrive bye eleven fifteen
i need to find a train that arrives at 12:30 on wednesday
i am departing from birmingham new street and going to cambridge
yes i need to arrive in cambridge bi twelve thirty please
i would like to go a on saturday
i need an trained leaving cambridge after seventeen thirty
i want to go to stansted airport on saturday
i am looking for a train from cambridge on monday
yes i need three seats on the train pleasee
i am looking for train that goes to london liverpool street and leaves after seventeen fifteen
i i want to leave on friday
i am looking for a train that departs from cambridge on thurs
i need to go to ily after twelve o 'clock
i will be leaving from cambridge
i need to go to birmingham new street
monday is the day uh i plan to travel
we just one ticket please
i want to lev from london liverpool street
i will depart on wednesday
i would like to find a a train that is leaving cambridge on monday
i 'll be leaving from london kings cross an need to leave after seventeen fifteen
i need to leave cambridge on sundae by train
i need to find a train that it 's going to arrive in stevenage by eleven forty five
i need to find the train going to stand stead airport and departing on thursday
i am looking for a train leaving from nore which
i am looking to go to cambridge en tuesday
i will be departing from norwich on wednesday
can you please book it for four people
i 'm going from cambridge to the stamps ted airport
and i would like to leave after thirteen forty five
uh i only need one ticket today
i want to get it rain to travel from cambridge to bishop stortford
just one ticket please
hello i need to arrive at london kings cross no later than 16:45
i 'm looking for a train pleasee going to cambridge
i 'm looking for a train leaving from london liverpool street after a twenty hundred hours on friday
i i 'm looking for a train departing from cambridge
i am looking for a train on sundae from cambridge
can you verify that it will arrive in stevenage bi nine o 'clock
i need n't trained departing from cambridge
ai will be going to london liverpool street on sunday
i 'm looking for a train going to cambridge plese
does that depart from broxbourne station
i want to leave en sunday
i would prefer to leave after 13:45 please
can i please get information for a train going from cambridge to broxbourne
can you give me that rain idea in price of the first train that departs after nineteen forty five
i 'm looking for a trained from cambridge to the stand said airport
i need n't rain to cambridge
around eight o 'clock and ip arting for cambridge
that it should work fine can i get tickets for six people
i need to train from cambridge to nor which please
i would like to depart on sunday
i need to arrive by 12:30
and i only want one ticket please
i would like to travel on sundays
uh i need to leave after seventeen fifteen
yes i need yt booked for two people
i 'd like a train that can get me from cambridge to birmingham new street on monday by twelve forty five
i am looking for a train that leaves on thursday and derives buy nine fifteen
i i 'm leaving from cambridge where i am now
i am looking for a train going to birmingham new street and should leave on thursday
iam departing from cambridge and want that rain to arrive by seventeen fifteen
i would like to leave cambridge and go-to london liverpool
i want to arrive by 15:45 on tuesday
i 'm looking for a train out of cambridge on saturday
i want to go to kingsland
i need it to find the train departing from stansted airport going to cambridge
i would like to go this saturday to came bridge
i 'm looking for a train going to e li please
it will be uh for thursday
i just need to arrive by 14:15
i am going to cambridge from broxbourne
i 'd like it to bee for saturday
i just need to leave bye a thirty
yes that would be fine plea 's book for six people
i am looking for a train from nor which on monday
the train should leave after 17:00 and should go to cambridge
i need to arrived by thirteen hundred hours
i need of booking for four people
i need to travel on wednesday and get there by 17:15
i would like to leave cambridge after nine fifteen
i would like to book a train for monday dip arting from kings linden arriving at cambridge at fourteen forty five
can you please provide me with the dip archer time of the train arriving by fourteen forty five or or just before
i am looking for a train to london kings cross that dip art 's after eight fifteen
i need help finding a train leaving cambridge aftr twenty forty five
i need to get it to london liverpool street
i would like to leave on thursday
i need a train on wednesday that leaves after 16:00
i am going to kingsland
it should arrive by nine forty five
i would like to leave an sundae
bta birmingham new street on wednesday
i need to get a train out of cambridge
the train should go to stand said airport and should arrive by twenty one thirty
i would like to leave on tuesday
whichever train goes to stamps to an airport tuesday from cap ridge arriving closest to twenty one thirty
i 'd like to train to heely that leaves wednesday
great can i book that for eight people an get a reference number please
i am looking for it rain
i am looking to go to london kings cross and depart from cambridge
i 'll be leaving on friday after 12:45
i 'd like to leave from bronx born some time after twelve thirty
i am trying to find a train leaving after fifteen fifteen to go to broxbourne
i am looking at travelling an sunday
can you help get tickets for a train on tuesday
i need to go to cambridge 's from stevenage after ten forty five
i need n't train on wednesday
i 'd like a train to london liverpool street that leaves after ten o 'clock
i will be leaving from cambridge en friday
i 'm looking for a trained departing from brooks borne
i i am heading to cambridge
i will need to arrive by 17:15
i will be leaving on wednesday
does that train are i 've in cambridge before seventeen fifteen
i need n't rain to cambridge
i want to leave me li on friday
uh i need to leave after nine forty five
uh i will be leaving saturday morning at eight forty five
sorry for the confusion and i need to get to peterborough thursday night
i need to leave after 20:15
and just one
i want to go from norwich to cambridge
i would like to arrive in cambridge by fourteen fifteen
i need a booking for two passengers
i 'd like to find that rain leaving on friday to cambridge
i 'm looking for a train from cambridge
i need to get to stanstead ther port please
i 'd like to travel on thursday
uh i need to leave after sixteen fifteen
i am looking to book a ticket for two people for the tuesday trained leaving from cambridge and arriving by ten thirty at broxbourne
i will be travelling on monday and need to arrive bye fourteen fifteen
and i would like to travel on monday after twelve o 'clock
i want to go to leicester
uh i need to go to cambridge on monday
just one please
i want to book a train that departs from cambridge and arrives in peterborough buy twenty one forty five
i i need to train from london liverpool street station on wednesday
i need to arrive by 14:45
i want to leave en wednesday
yes that works great can i book that for 5 people please
can i find a train to peterborough departing from cambridge
im going to kings lynn
uh i would be leaving from cambridge
they trains should leave after ten fifteen and should go to peterborough
i need to train from cambridge to healy
i 'm looking for a train it should arrive bi eighteen thirty and should go to peterborough
i need to leave on sunday from cambridge
i am looking for train going to cambridge please
i 'm looking for a train going to cambridge
i will be leaving from brooks hite
i want to go from london liverpool street to cambridge
i want to leave after 08:45 please
and i just need one ticket please
could you tell me when that reign arrives in london liverpool street
i actually need to levon saturday
i need a train that leaves this thursday from cambridge to yearly
i need to leave after seventeen forty five so anytime after that would be great
iam looking for a train from cambridge to lester
ai will be leaving on monday
i 'd like to find a a train to peterborough
can you reserve sick stick it 's on the next train after eleven fifteen
i would like to leave after the nine o 'clock on sunday
uh i am going to cambridge
they will be leaving from london liverpool street
i need on that will leave london after 21:00
i need to leave from cambridge and get to kings lin by eight o 'clock on saturday
looking for a train going to cambridge
i am leaving on sunday and need to arrive by thirteen hundred hours
i 'm looking for a trained going to brooks born please
i 'm looking for a train that will be going to london kings cross en wednesday
i need it to arrive by eighteen hundred hours
please find a train that dip parts from peterborough and arrives in cambridge by eight forty five
so i would like to leave on saturday from london liverpool street
i am looking for a train schedule that dip parts from cambridge and should go to healy
can you book that for 3 people and get me a reference number as well please
i need to book the train
i 'm at bishop stortford an would like to go to cambridge
i would like to travel this wednesday and need to leave after eight fifteen
would you be able to find me a train leaving on sunday
i would like to go on wednesday
i really must arrive by a eighteen fifteen to make my meeting
i i will be going from cambridge to broxbourne
i 'm looking for a train to take five meter came bridge
i would like to leave from stevenage on thursday and gett to cambridge by nine fifteen
i am leaving leicester and i need to leave by 20:30
i needed train on tuesday out of kings lynne
i want a leave on tuesday
looking for a trained tuesday leaving kings lynne after nine thirty to go to cambridge
i need a train out to london liverpool street on saturday
i need help finding a train leaving cambridge after 15:30
i am going to cambridge
i 'll be going to peterborough from cambridge
yes pleasee book it for one person and provide the reference number
i am looking for a train leaving after eleven forty five
i will be leaving stanstead their port friday two head tord cambridge
that would work for me can i get a trained for seven people
i would like to go to birmingham new street and the train should leave after 14:15
i am looking for a train that goes to a cambridge around ten forty five
i need to get a train from cambridge to norwich
are there any trains leaving from cambridge en thursday
i i 'm heading to broxbourne
i am looking for one that will arrive by 15:00
i need that for six people
i i need to train to peterborough from cambridge on sunday
i would like to leave after 18:15 please
i will be departing from norb which
so i would like to leave after fourteen thirty
i i need to book a train departing from cambridge after nine fifteen
i am looking for a train departing from london liverpool street on thurs to go to cambridge
i need a train to cambridge
i am looking for a train from kings limn
i would like to travel on on thursday
i 'd like to go after ten oclock
i am looking for it rain
i need to live on thursday from cambridge
i am looking for a train to peter boro
i am looking to go from cambridge to peterborough and leave after 14:15
i 'll be traveling on tuesday
i need a train going from stevenage to cambridge
i would like to leave on tuesday and kneed to arrive by nine o 'clock
i want to leave for a cambridge by train on wednesday
i 'll be coming from london kings cross
i i am looking for a trained departing from london kings cross
i would like to arrive in cambridge on tuesday by seventeen hundred hrs
i 'm leaving london liverpool street on sunday after twenty thirty
there are none closer to a 20:30 departure
i need to leave sunday to go to healy
yes can i get a ticket furet people please
i need to please
does that train leve on friday
thank you
i need a train going from ely to cambridge please
i 'd like to leave on friday and derive buy fourteen forty five
i i 'm going from stevenage to cambridge
yes i would like to book it for 8 people and i will need the reference number
i want to leave from stevenage on saturday
i would like to leave after twelve fifteen uh
i will depart any time so long as the train arrives by 20:45
i need to make a train reservation for friday and i need it to arrive buy nineteen hundred hours
i will be departing from cambridge 's
i will be going to stand steady airport
the train should go to cambridge and should ip art from london liverpool street
i need to train to take me from cambridge to brocks born
i will be traveling on monday
can i get a train from cambridge to bishops tort ferb
anh thursday after twenty fifteen
i am looking for a train that departs cambridge after twenty one hundred hours on thurs
i am looking for a train to cambridge en saturday
uh i would like to travel on saturday
i 'd like to arive by thirteen forty five
i need a train that leaves on wednesday with an arrival of 12:30
im traveling from cambridge to the london liverpool street
i 'd like to find a train tuesday adi parting from lester
i would like to leave aftr eleven fifteen
yes i would like to go from lester to cambridge and leave after eleven fifteen pleas
i 'd like to find a train from lysator on saturday
i am going to cambridge and need to leave after twelve o 'clock
i am looking for a train from london kings cross aan leaves on wednesday
i need a train leaving from london liverpool street on friday
i need to arrived by fourteen hundred hours
i 'm looking for a train from norwich to cambridge
i would like to leave an tuesday and arrive by thirteen thirty
i am looking for a train on friday that leaves after twenty one forty five
i will be leaving from cambridge and i want to go to stanstead airport
i need to leave after 21:45
i will be departing from london kings cross after 19:00
i need it for 6 people
i am looking for a train leaving london kings cross and deriving buy fourteen thirty
so i 'm going to cambridge
yes please book me something around 10 for 7 people
i will need to leave from cambridge
i would like to travel on a thursday
so i just need to arrive by seventeen forty five
i would like to catch a train leaving cambridge on sunday
i need to get to leicester on sunday
i want to leave after ten or fifteen
i am looking for a train that leaves bishops stortford and goes to cambridge
i plan to lev this saturday
i would like to arrive in cambridge by 8:15
i need a train from london kings cross to cambridge
is there a trained scheduled for sunday
i 'm dot looking for a train dip parting from stevenage
and i need to go to cambridge on friday
i wanted to leave some time after thirteen thirty
