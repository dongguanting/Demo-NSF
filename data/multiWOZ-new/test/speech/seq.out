O O O O O O O O O B-day O O O B-arrive I-arrive I-arrive
O O O O O B-depart I-depart
O B-people O
O O O O O O O O I-depart O B-leave I-leave I-leave
O O O O O O O B-depart O B-leave I-leave I-leave
O O O O O O O B-day O
O O O O O O B-depart I-depart I-depart O O O
O O O O O O O O B-leave I-leave
O O O O O O O O B-dest
O O O O O B-depart I-depart I-depart O B-day
O O O O O O O B-arrive O
O O O O O O O B-people
O O O O O O B-dest O O O B-arrive
O O O O B-depart I-depart
O O O B-dest O B-depart O B-day
O O O O O O O O O B-leave I-leave I-leave
O O O O O B-arrive I-arrive O
O O O O O O O O B-depart I-depart I-depart
O O O O O O O B-dest O B-day
O O O O O B-dest O O O O B-arrive
O O O O O B-people O
O O O O O B-day O O O O O B-leave I-leave
O O O O O B-arrive I-arrive
O O O O O O O O O B-arrive O B-day
O O O O B-depart I-depart I-depart O O O B-dest
O O O O O O B-dest O B-arrive I-arrive O
O O O O O O O B-day
O O O O O B-depart O B-leave I-leave
O O O O O B-dest I-dest O B-day
O O O O O O O B-depart O B-day
O O O B-people O O O O O
O O O O O O O O B-dest I-dest I-dest O O O B-leave I-leave
O O O O O O B-day
O O O O O O O O O B-depart O B-day
O O O O O B-dest O O O O
O O O O O B-depart
O O O O O B-dest I-dest I-dest
B-day O O O O O O O O
O O B-people O O
O O O O O B-depart I-depart I-depart
O O O O B-day
O O O O O O O O O O O B-depart O B-day
O O O O O B-dest I-dest I-dest O O O O O B-leave I-leave
O O O O B-depart O B-day O O
O O O O O O O O O O O O O B-dest O B-arrive I-arrive I-arrive
O O O O O O O O B-dest I-dest I-dest O O O B-day
O O O O O O O O B-depart I-depart
O O O O O O B-dest O B-day
O O O O O B-depart O B-day
O O O O O O B-people O
O O O O B-depart O O B-dest I-dest I-dest
O O O O O O O B-leave I-leave I-leave
O O O O B-people O O
O O O O O O O O O B-depart O B-dest I-dest
O B-people O O
O O O O O O B-dest I-dest I-dest O O O B-arrive
O O O O O O O O O B-dest
O O O O O O O O B-depart I-depart I-depart O B-leave I-leave I-leave I-leave O B-day
O O O O O O O O O B-depart
O O O O O O O B-day O B-depart
O O O O O O O O B-dest O O O O
O O O O O O B-depart
O O O O O B-dest I-dest I-dest O O
O O O O O O O O B-dest O
O O O O B-depart O
O O O O O B-day
O O O O O B-leave I-leave O
O O O O O O O O O O B-depart O B-dest
O O O O O O O O O O O O O O O O B-leave I-leave O
O O O O O O O B-depart O O B-dest I-dest I-dest
O O O O O B-dest
O B-leave I-leave I-leave O O O O B-dest
O O O O O O O O O O B-people O
O O O O O B-depart O B-dest I-dest O
O O O O O O B-day
O O O O O B-arrive
O O O O B-people O O
O O O O O O B-day
O O O O O O B-leave I-leave
O O O O O O B-people O
O O O O O O O O O O B-depart O B-dest I-dest I-dest O B-day O B-arrive B-arrive B-arrive
O O O O O O O O O B-day O O O B-arrive I-arrive
O O O O O B-depart O O O O
O O O O O O O O B-dest I-dest I-dest O O O O B-day
O O O B-depart O O O O O O O B-arrive I-arrive
O O O O O B-depart O O O O
O O O O O O O B-day
O O O O O O O O B-depart O B-day
O O O O O B-dest
O O O O O O O O O B-depart I-depart O O B-dest
O O O O O O B-day O B-dest I-dest
O O O O O O O O B-dest I-dest O
O O O O O B-day
O O O O O O B-arrive
O O O O B-dest O B-depart
O O O O O O O B-day
O O O O O O B-leave I-leave
O O O O O O O O O B-people O
O O O O O O O B-depart I-depart O B-day
O O O O O B-leave O O O O B-dest
O O O O O B-arrive I-arrive I-arrive
O O O O O B-people O
O O O O O B-day O O O O B-arrive
O O O O O B-depart O B-leave I-leave
O O O O O O O O B-day O O O B-depart I-depart O O B-dest O B-arrive I-arrive I-arrive
O O O O O O O O O O O O O O O B-arrive I-arrive I-arrive O O O O
O O O O O O O B-dest I-dest I-dest O O O O O B-leave I-leave
O O O O O O O B-depart O B-leave I-leave I-leave
O O O O O O B-dest I-dest I-dest
O O O O O O B-day
O O O O O B-day O O B-leave I-leave
O O O O B-dest
O O O O B-arrive I-arrive I-arrive
O O O O O O B-day
O B-dest I-dest I-dest O B-day
O O O O O O O O B-depart
O O O O O B-dest I-dest I-dest O O O O B-arrive I-arrive I-arrive
O O O O O O B-day
O O O O B-dest I-dest I-dest I-dest B-day O O O O O O B-arrive I-arrive I-arrive
O O O O O O B-dest O O B-day
O O O O O O B-people O O O O O O O
O O O O O B-depart
O O O O O O B-dest I-dest I-dest O O O B-depart
O O O O O B-day O B-leave
O O O O O O B-depart I-depart O O O B-leave I-leave
O O O O O O O O O B-leave I-leave O O O B-dest
O O O O O O B-day
O O O O O O O O O B-day
O O O O O B-dest O O B-depart O B-leave I-leave I-leave
O O O O O B-day
O O O O O O B-dest I-dest I-dest O O O B-leave I-leave I-leave
O O O O O B-depart O B-day
O O O O O O O O B-depart I-depart
O O O O O B-dest
O O O O O O B-arrive
O O O O O B-day
O O O O O O O B-dest O B-arrive I-arrive
O O O O O B-dest
O O O O O B-depart O B-day
O O O O O O B-leave I-leave I-leave
O O O O O B-day O O B-leave I-leave I-leave
O O O O O O O O O O B-dest B-day O
O O O O O B-leave
O O B-people
O O O O O B-depart O B-dest
O O O O O O O O B-arrive I-arrive
O O O O O B-people O
O O O O O O O O O B-day O B-dest
O O O O O O O B-depart
O O O O O B-dest I-dest I-dest O
O O O O O O B-day
O O O O O O B-leave I-leave
O O O O O O O O B-people O O O B-day O O O B-depart O O O B-arrive I-arrive O B-dest
O O O O O B-day O O O O O B-arrive I-arrive
O O O O O O O B-day O B-leave I-leave I-leave
O O O O O B-dest
O O O O O O B-dest O B-day
O B-people O
O O O O O O O O O B-depart O O O B-dest O B-arrive I-arrive I-arrive I-arrive
O O O O O O B-depart I-depart I-depart O O B-day
O O O O O B-arrive
O O O O O B-day
O O O O O O O O O B-people O O
O O O O O O B-dest O O B-depart
O O O B-dest I-dest
O O O O O O B-depart
O O O O O B-leave I-leave O O O O B-dest
O O O O O B-depart O B-dest
O O O O O O O O O O B-arrive I-arrive O O O O B-dest
O O O O O B-day O B-depart
O O O O O O O B-dest O
O O O O O O O O B-dest
O O O O O B-depart I-depart
O O O O O B-depart I-depart I-depart O B-dest
O O O O O B-leave O
O O O O B-people O O
O O O O O O O O O B-dest I-dest O
O O O O O B-day
O O O O O O O B-day O B-depart O B-dest
O O O O O B-leave I-leave I-leave O O O O O O O
O O O O O O B-depart O B-dest
O O O O O B-day
O O O O O O O O O B-dest
O O O B-people O O O O O O O O B-leave I-leave
O O O O O B-leave I-leave I-leave I-leave I-leave O B-day
O O O O O B-dest
O O O O O B-depart I-depart I-depart
O O O O O O O O B-leave
O O O O O B-depart O O O B-dest I-dest O O O O O B-day
O O O O O O B-depart
O O O O B-day O O O O O B-arrive I-arrive I-arrive
O O O O O O O O B-dest I-dest O
O O O O O O O O O O O B-dest I-dest I-dest O B-day
O O O O O O B-arrive I-arrive I-arrive
O O O O O O O O B-depart O O O B-dest O B-arrive I-arrive I-arrive
O O O O O O O B-day O O O O
O O O O O O O O O O O B-depart O O O O B-dest
O O O O O B-people O O O O O O O O O O
O O O O O B-dest
O O O B-depart I-depart O O O O O O B-dest
O O O O O O B-day O O O O O B-leave I-leave
O O O O O O O O O O O B-day
O O O O O O B-day
O O O O O B-arrive I-arrive I-arrive O O O O
O O O O O O B-depart O B-dest
O O O O O O O O O O B-dest I-dest
O O O O O O B-depart O B-day O O O O O O O
O O O B-depart O O O O O O B-leave
O O O O B-day O O B-depart I-depart
O O O O O B-day
O O O O B-day O B-depart I-depart O O O O O O B-dest
O O O O O O B-dest I-dest I-dest O B-day
O O O O O O O B-depart O B-leave
O O O O B-dest
O O O O O B-dest O B-depart
O O O O O B-people O O O O O O
O O O O O O O O B-leave I-leave I-leave
O O O O B-depart I-depart I-depart B-day O O O B-dest
O O O O O O O O O O O B-people O
O O O O O O B-dest I-dest I-dest O O O O O O B-leave
O O O O O O O O O O B-dest O B-arrive I-arrive I-arrive
O O O O O O O B-depart O B-dest
O O O O O O B-depart O B-day
O O O O O B-dest
O O O O O O O O O B-arrive
O O O O B-people O
O O O O O O B-dest O B-depart O B-day
O O O O O O B-leave O
O O O O O B-depart I-depart
O O O O O O O B-leave I-leave
O O O O O O O O O B-depart O O O
O O O O O O O O B-depart I-depart I-depart O O O O O B-dest
O O O O O B-dest
O O O O O O O B-depart I-depart
O O O O O O O B-day
O O O O O O B-leave I-leave
O O O O O B-depart
O O O O O B-day O B-depart
O O O O O O O B-dest I-dest
O O O O O O B-depart O O O O O B-leave
O O O O O B-day
O O O O O O B-depart O B-dest
O O O O O O B-day O O O O O B-arrive I-arrive I-arrive
O O O O O O B-dest O O O B-day
O O O O O B-depart I-depart I-depart
O O O O O O O O O B-depart I-depart I-depart
O O O O O O B-dest O B-day O B-arrive I-arrive I-arrive
O O O B-depart I-depart I-depart O B-day O B-leave I-leave
O O O O O O B-leave O
O O O O B-day O O O B-dest
O O O O O O B-people O O
O O B-people O
O O O O O B-day
O B-dest
O O O O O O B-depart O B-dest O
O O O O O O B-day O O O B-arrive I-arrive I-arrive
O O O O O B-depart O B-dest
O O O O O O O O B-people O O O O O O O O
O O O O O B-depart O B-day
O O O O O O B-leave I-leave O
O O O B-leave O O O O O O O O B-arrive
O O O O O O O O B-day O O O O O O O B-arrive I-arrive I-arrive
O O O O O B-depart O
O O O O O B-dest I-dest I-dest
O O O O O B-dest O O O O O B-depart I-depart I-depart
O O O O O O O O B-depart O B-dest I-dest
O O O O O B-day
O O O O O O B-depart O B-dest I-dest I-dest
O B-day O B-leave I-leave
O O O O O O O O B-depart O B-leave I-leave I-leave I-leave O B-day
O O O O O O O B-dest O B-day
O O O O O O O B-day
O O O O O O B-arrive I-arrive I-arrive
O O O O O O O B-day O O O O B-arrive
O O O B-depart O O B-dest I-dest I-dest
O O O O O O O B-day O O O B-depart
O O O O O O B-leave I-leave
O O O O O O O O O B-dest O O O O O O
O O O O O O O O O O B-day
O O O O B-dest O O O O O B-leave I-leave I-leave
O O O O O O O B-depart I-depart I-depart O O O B-day
O O O O O O B-depart I-depart I-depart O O
O O O O O B-arrive I-arrive I-arrive
O O O O O O O B-depart O B-dest
O O O O O O B-day O O O B-arrive I-arrive
O O O O O O O B-day O O O B-leave I-leave I-leave I-leave
O O O O O B-depart O O O O O O B-dest I-dest
O O O O O B-leave
O O O O O B-depart I-depart I-depart O B-leave
O O O O B-people O
O O O O O O O B-depart I-depart I-depart O O O B-arrive I-arrive
O O O O O B-dest
O O O O O O B-leave O B-people O
O O O O O O B-depart
O O O O O O O B-day
O O O O O O O B-arrive I-arrive I-arrive
O O O O O O O O B-depart O B-day
O O O O O B-dest O O
O O O O O B-leave I-leave I-leave
O O O O O O O O B-depart I-depart O O O B-dest
O O O O O B-day
O O O O O O O O B-arrive
O O O O O B-depart I-depart I-depart O B-dest
O O O O O O B-day
O O O O O O O O O O B-depart
O O O O O O B-dest O B-day
O O O O O O O B-leave I-leave
